processor/src/main/kotlin/com/tobrun/datacompat/DataCompatProcessor.kt

```diff
@@ -36,14 +54,15 @@ class DataCompatProcessor(
 
     private inner class Visitor : KSVisitorVoid() {
 
+        @Suppress("LongMethod")
```

Comment:
:sweat_smile: once I have the MVP fully functional I'm planning to figure out a nice division in classes. I would call it an architecture but it needs cleaning up.